api/storage/elasticsearch/elasticsearch.go

```diff
@@ -501,3 +513,40 @@ func (sm *StorageManager) createIndex(index string) {
 	log.WithField("index", index).Info("index created successfully")
 
 }
+
+// checkCurrentIndexDay is charge of getting the duration time until date change (2020-01-01 23:00:00 -> 2020-01-02 00:00:00).
+// when the day changed, the function call to `setCurrentIndexDay` for create new index with current date
+func (sm *StorageManager) checkCurrentIndexDay(now time.Time) {
```

Comment:
I would simplify this function (name and functionality), something like; waitUntilTomorrow and won't set the index name here, WDYT?

Reply:
changed.